### ISSUE TITLE
Theme settings set in Admin saved under user/env/.../config/themes instead of user/config/themes

### ISSUE THREAD

**Issue by yankl** [getgrav/grav#3497]
In the documentation it says: 

> By providing a correctly-formatted blueprints.yaml file, you can allow the user to change the settings from the Admin interface. When the settings are saved, they're automatically written to /user/config/plugins/archives.yaml (or under config/themes, if it's a theme).
(https://learn.getgrav.org/17/forms/blueprints/example-plugin-configuration)

However, after setting up my theme's blueprints.yaml and saving the settings in the Admin interface, the new config file is saved under env/localhost/config/themes (as I'm still developing my theme/site locally). 

Is this intended behavior? Does this always happen? Is there any way to get settings to be saved under user/config? The particular settings here were of the "filepicker" sort.. don't know if that makes a difference.

**Comment by mahagr** [comment]
Yes, this is the behavior of admin and yes, it is not optimal...

The only way to make it to save the file into the main folder is to either not use the environment or create the file into the main configuration and delete the override.
